tests/functional/Tribe/Events/Cost_UtilsTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+namespace Tribe\Events;
+
+class Cost_UtilsTest extends \Codeception\TestCase\WPTestCase {
+
+	public function setUp() {
+		// before
+		parent::setUp();
+
+		// your set up methods here
+	}
+
+	public function tearDown() {
+		// your tear down methods here
+
+		// then
+		parent::tearDown();
+	}
+
+	/**
+	 * @test
+	 * it should be instantiatable
+	 */
+	public function it_should_be_instantiatable() {
+		$sut = $this->make_instance();
+
+		$this->assertInstanceOf( 'Tribe__Events__Cost_Utils', $sut );
+	}
+
+	public function merge_cost_ranges_into_ranges() {
```

Comment:
Is it worth covering values with a decimal point (and different formats of decimal points) and should we cover use of thousands separators?